charts/kubescape-prometheus-integrator/values.yaml

```diff
@@ -7,6 +7,12 @@ createKubescapeServiceAccount: true # TODO: move to kubescape
 # -- set the image pull secrets for private registry support 
 imagePullSecrets: ""
 
+# -- list of namespaces to include
```

Comment:
@cgroschupp This value should be in the Kubescape microservice scope since it applies only to the KS microservice